CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ We welcome feedback, bug reports, and pull requests!
 For pull requests (PRs), please stick to the following guidelines:
 
 * Before submitting a PR, verify that [an issue](https://github.com/rotki/rotki/issues) exists that describes the bug fix or feature you want to contribute. If there's no issue yet, please [create one](https://github.com/rotki/rotki/issues/new/choose).
-* Fork rotki on your GitHub user account, make code changes there, and then create a PR against develop rotki repository.
+* Fork rotki on your GitHub user account, make code changes there, and then create a PR against the develop rotki repository.
```

Comment:
Hey @mdqst thanks! Can you change it to
```suggestion
* Fork rotki on your GitHub user account, make code changes there, and then create a PR against the develop branch of the rotki repository.
```

Reply:
[LefterisJP](https://github.com/LefterisJP), updated.